### ISSUE TITLE
API proposal: get_stream() to get OpenTok stream information

### ISSUE THREAD

**Issue by normanargueta** [opentok/Opentok-Python-SDK#129]
This is a proposal to add a new OpenTok Python SDK method for getting stream info.

### opentok.get_stream()

This method returns an Stream object that contains information of an OpenTok stream as is described in the [documentation](https://tokbox.com/developer/rest/#get-stream-info).

### Usage
```python
session_id = 'SESSIONID'
stream_id = '8b732909-0a06-46a2-8ea8-074e64d43422'

# To get stream info:
stream = opentok.get_stream(session_id, stream_id)

# To get stream info from a Session instance:
stream = session.get_stream(stream_id)

# Stream properties:
print stream.id #8b732909-0a06-46a2-8ea8-074e64d43422
print stream.videoType #camera
print stream.name #stream name
print stream.layoutClassList #['full']
```

**Comment by msach22** [comment]
Thanks @normanargueta! This looks great!

**Comment by msach22** [comment]
This has been released with 2.7.0
